data/json/recipes/recipe_armor.json

```diff
@@ -8339,17 +8339,63 @@
       [ "hat_hunting", 1 ]
     ],
     [
+      [ "mask_dust", 1 ],
```

Comment:
Please revert this part, it's described as just a cowl, so you're not tweaking it here but making it a different thing.  The extra protection too of course.